osmoutils/store_helper.go

```diff
@@ -138,10 +138,9 @@ func MustGetDec(store store.KVStore, key []byte) sdk.Dec {
 	return result.Dec
 }
 
-// GetIfFound returns a value at key by mutating the result parameter. Returns true if the value was found and the
-// result mutated correctly. False otherwise. Returns error only when database or serialization errors occur
-// returns a boolean indicating whether value exists for the given key and error
-func GetIfFound(store store.KVStore, key []byte, result proto.Message) (found bool, err error) {
+// Get returns a value at key by mutating the result parameter. Returns true if the value was found and the
+// result mutated correctly. False otherwise. Returns false and error only when database or serialization errors occur
```

Comment:
small nit: it was a bit difficult to read
```suggestion
// Get returns a value at key by mutating the result parameter. Returns true if the value was found and the
// result mutated correctly. Returns False otherwise and errors only when database or serialization errors occur
```

Reply:
Err, this actually changes the semantic meaning!

Reply:
Made another suggestion for what I think you intended

Reply:
Right on point!